### DISCUSSION TITLE
Where is the extension's installation address on linux?

### DISCUSSION THREAD

**Discussion by leijue222** [coder/code-server#5376] [Q&A] [answered]
I want to package the installed plug-ins together, so that I can easily unzip and install them before several machines, instead of reinstalling the plug-ins every time.

**Comment by flxcp** [comment, marked as answer]
All the extension are stored in `~/.local/share/code-server/extensions`.

You can check the [docs](https://coder.com/docs/code-server/latest/FAQ#where-are-extensions-stored) for more information.
